eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/components/soap/SOAPAuthenticationComposite.java

```diff
@@ -254,47 +254,47 @@ private void updateControls(boolean isBasic, boolean isNtlm) {
     
     private void bindControls(final DataBindingContext context) {
         final EditingDomain domain = AdapterFactoryEditingDomain.getEditingDomainFor(getTargetObject());
-        final Realm realm = SWTObservables.getRealm(_authTypeCombo.getCombo().getDisplay());
+        final Realm realm = DisplayRealm.getRealm(_authTypeCombo.getCombo().getDisplay());
 
-        _bindingValue = new WritableValue(realm, null, SOAPBindingType.class);
-        final IObservableValue authType = new WritableValue(realm, null, String.class);
-        final IObservableValue basicAuthUser = new WritableValue(realm, null, String.class);
-        final IObservableValue basicAuthPwd = new WritableValue(realm, null, String.class);
-        final IObservableValue ntlmAuthDomain = new WritableValue(realm, null, String.class);
+        _bindingValue = new WritableValue<>(realm, null, SOAPBindingType.class);
+        final IObservableValue<String> authType = new WritableValue<String>(realm, null, String.class);
```

Comment:
I mentioned it only on previous line before but you can write new WritableValue<String> also for all the following lines.